packages/circuit-ui/components/ComparisonTable/components/PlanTable/PlanTable.tsx

```diff
@@ -108,28 +107,32 @@ export const PlanTable = forwardRef<HTMLTableElement, PlanTableProps>(
         0,
       ) > COLLAPSE_THRESHOLD,
     );
-    const [headerHeight, setHeaderHeight] = useState(0);
+    const [sectionOffset, setSectionOffset] = useState(0);
     const isPlanPickerVisible = headers.length > 2;
-    const offset =
-      headerHeight +
-      (isPlanPickerVisible ? (isMobile ? 80 : 0) + (isTablet ? 16 : 0) : 0);
-
-    const updateHeaderHeight = useCallback(() => {
-      throttle(() => {
-        setHeaderHeight(theadRef.current?.getBoundingClientRect().height ?? 0);
-      }, 500)();
-    }, []);
 
     useEffect(() => {
-      updateHeaderHeight();
-    }, [updateHeaderHeight]);
+      if (!theadRef.current || !ResizeObserver) {
```

Comment:
An undefined variable is different from an uninitialized one. The current approach would throw the following error:

```
ReferenceError: ResizeObserver is not defined
```

Use the `typeof` operator to safely check whether a variable exists/has been initialized:

```suggestion
      if (!theadRef.current || typeof ResizeObserver !== 'undefined') {
```

Reply:
you mean 
```suggestion
      if (!theadRef.current || typeof ResizeObserver === 'undefined') {
```
?